docs/development/design/new-sphinx-guides.rst

```diff
@@ -0,0 +1,186 @@
+Proposed contents for new Sphinx guides
+=======================================
+
+This classification follows the `Diátaxis Framework <https://diataxis.fr/>`__.
+
+Tutorial
+--------
+
+   **Objective**: Write a tutorial, some examples, and reference
+   documentation for our Python code.
+
+Every step should lead to a “yay!” moment, which marks the end of the
+section. This keeps the learner motivated.
+
+Appendixes are optional, i.e. not required to follow the tutorial, but
+highly recommended.
+
+1. Getting started
```

Comment:
This seems useful, but I don't know if we want to have so much introductory content in our tutorial. This seems like something we can link out to existing docs/tutorials for. I feel like we want to maintain content focused on Sphinx, and external sources will be better canonical references for virtualenv, adding dependencies, etc. 

We could consider including some of this, but I'd want to do this last if we have user requests, instead of first.

Reply:
I was also hesitant to introduce it, but [as I said above](https://github.com/readthedocs/readthedocs.org/pull/8106#discussion_r614136579), having a short preparation like this would make the tutorial more consistent, I think. Otherwise, it's might end up being a series of random sentences, "lorem ipsum"-like text, or stock photos, which is what many other Sphinx tutorials do.

For example, Brandon tutorial https://brandons-sphinx-tutorial.readthedocs.io/en/latest/ creates a fictitious library called `trianglelib`. It omits lots of details because it's meant for a 3-hour PyCon tutorial, but we can afford to do much more.

Reply:
Greatly simplified the introduction.